Makefile

```diff
@@ -193,3 +197,11 @@ test/k3s-%: dist/img.tar bin/k3s dist-%
 
 .PHONY: test/k3s/clean
 test/k3s/clean: bin/k3s/clean;
+
+.PHONY: clean
+clean:
+	rm -rf dist
+	rm -rf bin
+	$(MAKE) test-image/clean
+	$(MAKE) test/k8s/clean
+	$(MAKE) test/k3s/clean
```

Comment:
can we prefix these with `-` so that we keep going if they fail (e.g., in case they were not installed)?
https://www.gnu.org/software/make/manual/html_node/Errors.html